### ISSUE TITLE
some issues feedback to you

### ISSUE THREAD

**Issue by bau720123** [initialxy/cordova-plugin-themeablebrowser#11]
hi @initialxy 
I am a PGB's user (build.phonegap.com)

I use your full code at here (really as same as yours)
https://github.com/initialxy/cordova-plugin-themeablebrowser

and...this is my screenshot (left side is android 4.4.2，the right side is ipad mini iOS 8.2)
http://www.littlebau.com/th.jpg

issue 1
no mater Android or iOS，The upper left and right corner's "button" disappear

issue 2
in android，when I click..
upper left corner，it will "back" to the app
upper right corner，it will show the "menu"，and click 'Hello World!' will trigger it's behavior
but in iOS...it will no any response

**Comment by initialxy** [comment]
Hi, thanks for trying this out. Your issue sounds like it didn't link the button images correctly. So the buttons are there (and seemingly working), but button image icons are not shown. Please verify that the images you linking are available in **native** resource.

One thing I do like to point out is that if you imply installed this plugin and copied and pasted the sample code on [README.md](https://github.com/initialxy/cordova-plugin-themeablebrowser/blob/master/README.md), it is expected to have a blank look, which is explained in [here](https://github.com/initialxy/cordova-plugin-themeablebrowser/blob/master/README.md#i-just-installed-this-plugin-how-come-it-just-shows-a-blank-toolbar) in [README.md](https://github.com/initialxy/cordova-plugin-themeablebrowser/blob/master/README.md) as well as the comment on top of the sample code. The reason is that as a Cordova plugin, I don't want to pollute your resource bundle with bunch of images, which you probably not gonna use, because the whole reason why you installed this plugin is to use your own button icons. That being said, the images in the sample are in the [source](https://github.com/initialxy/cordova-plugin-themeablebrowser/tree/master/src/ios/Resources) of this project, it's just that they are not imported to your project during plugin installation and are left there just for dev and testing purposes.

Please ensure that this is not the case.

**Comment by bau720123** [comment]
hi @initialxy 
I know you meaning
but in PGB => everything is" cloud build" and automatically
there is no way to add some "Resources" Manually
if could...let your plugin can use outside resource maybe a good idea
just like that
```
backButton: {
        image: 'http://somedomain/sompic.png',
    },
```

**Comment by initialxy** [comment]
Hi @bau720123, thanks for the feedback, however that poses a whole new challenge that is outside of the concern of this plugin. In fact, retrieving a native image from the web dynamically will not work well at all in native, because native handles image resources differently depending on pixel density, and it's handled differently from platform to platform. [This guide](https://github.com/initialxy/cordova-plugin-themeablebrowser/blob/master/README.md#import-native-images) goes into a bit of explanation. It would be difficult and messy to display them with their intended size on different pixel densities.

I don't know how your continuous integration build system works, however I'd recommend setting up your build script to copy image resources into their corresponding directories and update xcodeproj accordingly as a pre-build procedure. In fact, that's what I'm currently doing for a project I'm working on. The alternative approach I've done before is to modify the xcodeproj file a head of time and save it somewhere in the build source, then have a build script copy and overwrite the generated xcodeproj with the one you've modified, along with all the images.

**Comment by initialxy** [comment]
Hi @bau720123, out of curiosity, I've read [build.phonegap.com](https://build.phonegap.com/)'s documentation, and it seems like it's super strict on their own ideology of build system that it doesn't give its users much freedom at all.

That being said, I'd commend the following. Since ALL you have to do is import some image sources right? Something like that can be easily achieved through installing a plugin. In fact when you import images during plugin installation, you don't even need to worry about modifying xcodeproj, Cordova will take care of that for you. I've also found out that PhoneGap Build now [allows **private** plugin](http://phonegap.com/blog/2014/12/09/phonegap-build-new-features/) (see last item). So I'd recommend you to make a private plugin, and the only thing it ever does is import bunch of images to native projects. You don't even need to write ANY native code, you just need a stub plugin setup and add the resource import tags into [plugin.xml](https://cordova.apache.org/docs/en/4.0.0/plugin_ref_spec.md.html). Then submit it to https://build.phonegap.com/plugins as a private plugin, and include this plugin in your project.

For an example, see an older version of [plugin.xml](https://github.com/initialxy/cordova-plugin-themeablebrowser/blob/018a0d8acd00e01c7e693c731707032bf648d017/plugin.xml) on this repo. Notice the `resource-file` tags. That's all you need.

**Comment by initialxy** [comment]
See [experimental features](https://github.com/initialxy/cordova-plugin-themeablebrowser#experimental-properties) available in version 0.2.10, which is already migrated to npm with respect to issue #23, so be sure to use the new ID, `cordova-plugin-themeablebrowser`.
